### DISCUSSION TITLE
Error creating PCH

### DISCUSSION THREAD

**Discussion by SrBagel** [FunkinCrew/Funkin#2757] [Compiling help] [answered]
Compiling results in this error: 
Error cleaning PCH file funkin/export/release/windows/obj/obj/msvc1964-nc/__pch/haxe/hxcpp.pch: std@file_delete
Error: Could not create PCH
I've followed the compiling guide and I don't know why this is happening

**Comment by SrBagel** [comment]
If it helps I am using the latest version of Haxe (4.3.4)

**Comment by NotHyper-474** [comment]
Have you tried doing a clean build already? (i.e`lime test windows -clean`)

**Reply by SrBagel** [reply]
Okay. Now I'm getting this error: funkin/.haxelib/flixel-addons/3,2,3/flixel/addons/tile/FlxTilemapExt.hx:337: lines 337-427 : Warning : overlapsWithCallback is deprecated, use objectOverlapsTiles
source/funkin/graphics/shaders/RuntimeCustomBlendShader.hx:14: characters 10-23 : funkin.graphics.shaders.RuntimeCustomBlendShader has no field setBitmapData

**Reply by NotHyper-474** [reply]
Maybe try reinstalling the libraries?

**Reply by SrBagel** [reply]
Still isn't working its fine though all I did was add a setting that lets you cap your fps so I'll just wait until they add that thanks for the help anyway

**Reply by SrBagel** [reply]
There is also a chance that my antivirus is fucking things up (it loves to do that)

**Comment by SrBagel** [comment, marked as answer]
Okay. I reinstalled the libraires with `hmm reinstall -f` and it was my antivirus fucking with things, so I fixed that and everything is working now!

**Reply by NotHyper-474** [reply]
Very nice!
